src/validate.js

```diff
@@ -30,12 +29,12 @@ var resolutionSchema = exports.resolutionSchema = Joi.alternatives().try(
 
 var browsersSchema = exports.browsersSchema = Joi.array().min(1).unique().items(
   Joi.object().keys({
-    browserName: Joi.string().valid(['chrome', 'firefox']).required(),
+    browserName: Joi.string().valid('chrome', 'firefox').required(),
```

Comment:
@xwa130saucer why is this changing to be an array to a set of arguments?

Reply:
@vrunoa
You are right, it's just for `Joi`